hardhat.config.ts

```diff
@@ -95,9 +90,6 @@ const config: HardhatUserConfig = {
     BOT: {
       default: 2
     },
-    OPERATIONAL_TREASURY: {
-      default: 3
-    },
     TEST1: {
       default: 4
```

Comment:
```suggestion
      default: 3
```

Change also the counter of the accounts below